docs/toolset/integration.md

```diff
@@ -0,0 +1,180 @@
+# Integration data model
+
+The integration JSON payload contains the complete information sample that an integration sends through the New Relic
+Infrastructure Agent, at a given moment.
+
+This document describes the basic structure and elements of the integration payload, as well as the basic API by means
+of simple examples.
+
+For detailed information about the `integration` package, please refer to the
+[Integration package GoDoc page](https://godoc.org/github.com/newrelic/infra-integrations-sdk/integration)
+
+## Building an integration
+
+To build an integration with the default configuration, you have to invoke the
+[integration.New](https://godoc.org/github.com/newrelic/infra-integrations-sdk/integration#New) function with
+the integration name and version as argument:
+
+```go
+payload, err := integration.New("my-integration", "1.0")
```

Comment:
IMO `payload` is not right naming as it's an object with behaviour, to keep up with Go semantics I'd use something like just `i`

Reply:
Good appreciation. I'll fix it in a new PR.